iphone/Classes/TiUIWebView.m

```diff
@@ -290,7 +290,10 @@ -(void)loadFile:(NSString*)absolutePath
 		textEncodingName:(NSString*)textEncodingName
 	   mimeType:(NSString*)mimeType
 {
-	NSURL *requestURL = [NSURL fileURLWithPath:absolutePath];
+    static unsigned long uniqueIdentifier = 0;
+    uniqueIdentifier++;
```

Comment:
Yes, already tried to add a key to the *NSURLRequest*, but for iOS they are anyway the same requests.
PS: I'm not a native ObjC/Swift developer, I only try to do my best, so I'm really sure that you'll see what I missed.